### DISCUSSION TITLE
Help? Any one come across a certain game that just won't with VDD?

### DISCUSSION THREAD

**Discussion by buthig666** [VirtualDrivers/Virtual-Display-Driver#211] [Q&A] [answered]
I have been using Sunshine/win11/13700k/4080/VDD host for a quite a while now, with various moonlight clients 1080-4k 60-120Hz.  I use this set up daily for work and gaming, in and out of lan.  I run moonlight exclusively in desktop mode. My Vdd (current Beta) and my monitor run mirrored. Everything has been functional and mostly stable.  

 Assassin's Creed Mirage won't open correctly or crashes when streaming with the VDD exclusively.   Went to test it in front of my actual host and mon, and it opens/plays fine 4k 120Hz.  The stream is fine and as stated the VDD is mirrored to the irl display so the desktop stream itself is fine.  When the monitor is on the game opens up and streams fine.  When the monitor is off the game won't open but the stream is still fine.  I can open and close any app I want as usual (I almost am never Infront of the actual Host.)

Never seen anything like this.  Is it an issue with this particular game?  Anyone else run into this?

**Comment by bud3699** [comment, marked as answer]
We're aware that some people are having these issues but the only problem is, i can't replicate it on my computer, the feature levels work fine, which makes me think it's power management on later cards, I'm running a 20 series cards

**Reply by buthig666** [reply]
Thanks for the response Bud.  I have played dozens of games with this setup and use the VDD daily for work tasks.  Vdd has been running rock solid since the Dec 2023 update.  So weird.  I was thinking it had more to do with something ubisoft is doing as my 4080 doesn't seem to mind the VDD at all except for this one game.
Oh well.  I thought I would ask before giving up on it.  Thanks for all the work you guys have done. Happy holidays to ya!
